gr-digital/python/digital/qa_ofdm_carrier_allocator_cvc.py

```diff
@@ -208,6 +208,32 @@ def test_003_t (self):
                 self.assertEqual(correct_offsets[key], tag.offset)
         self.assertTrue(all(tags_found.values()))
 
+    @gr_unittest.unittest.skip("Skipping test with wrong input (caused SIGFPE earlier, now throws a simple std::invalid_argument with useful message)")
```

Comment:
Instead of skippping this test, could you call the constructor with invalid arguments and then catch the correct exception? Also, are these arguments interdependend? This should be checked in the constructor as well. Then write some tests that try to call the constructor with invalid arguments and check if the appropriate exception is thrown.
`unittest.TestCase.assertRaises` seems to be the correct check here.